homeassistant/components/zha/light.py

```diff
@@ -1183,7 +1183,9 @@ def __init__(
         if self._zha_config_group_members_assume_state:
             self._update_group_from_child_delay = ASSUME_UPDATE_GROUP_FROM_CHILD_DELAY
         self._zha_config_enhanced_light_transition = False
-        self._attr_color_mode = None
+
+        self._attr_color_mode = ColorMode.UNKNOWN
```

Comment:
we should probably update `async_added_to_hass` to get this from restore_state for group entities too. Can be a separate PR if ya prefer.

Reply:
could be as simple as adding this to `async_added_to_hass`:

```
if last_state := await self.async_get_last_state():
    self.async_restore_last_state(last_state)
```

and then move `async_restore_last_state` from Light to BaseLight... Haven't tested - just took a quick peek.

Reply:
since the state is ultimately calculated from member states this probably isn't needed (could be why it wasn't done previously too).